deepseg.cc

```diff
@@ -110,16 +131,124 @@ void *grab_thread(void *arg) {
 	return NULL;
 }
 
+void init_tensorflow(calcinfo_t *info) {
```

Comment:
Probably should be reference, not pointer.

Reply:
done